src/tseda/datastore.py

```diff
@@ -423,10 +443,47 @@ def check_data_modification(self):
     def reset_modification(self):
         self.data.rx.value.sample_set_id = self.data.rx.value.population
 
+    def combine_tables(self, individuals_table):
+        """Combine individuals and sample sets table."""
+
+        combined_df = pd.merge(
+            individuals_table.rx.value,
+            self.sample_sets_table.data.rx.value,
+            left_on="sample_set_id",
+            right_index=True,
+            suffixes=("_indiv", "_sample"),
```

Comment:
Again, I would suggest using `_sample_set`, otherwise I think of a single sample (like an individual)

Reply:
Agreed! I will change that but will keep it as suffixes as pandas merge does not give the option to use prefixes